### PR TITLE
Reference added to error table

### PR DESCRIPTION
**Pull request by iArchitSharma**
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d555936cd86ef7d384c7e78dc2811f08d45047ef at: https://65cb9c0239023509852cb798--meshery-docs-preview.netlify.app

File: docs/pages/reference/error-codes.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Note: The numeric portion of error codes are component-scoped. The numeric porti
       <th style="width:15%">Severity</th>
       <th class="error-name-code"><span>Error Name - Code</span></th>
       <th style="width:85%">Short Description</th>
+      <th>Reference</th>
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:147]:
```suggestion
      <th>Discussion</th>
```

File: docs/pages/reference/error-codes.md [hunk 2]
```diff
@@ -161,6 +162,7 @@ Note: The numeric portion of error codes are component-scoped. The numeric porti
           <code>{{ err_code[1]["name"] | xml_escape }}-{{ err_code[1]["code"] }}</code>
         </td>
         <td>{{ err_code[1]["short_description"] | xml_escape }}</td>
+        <td><a href="https://discuss.layer5.io/search?q={{ err_code[1][code] }}" target="_blank">Visit</a></td>
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:165]:
```suggestion
        <td><a href="https://discuss.layer5.io/search?q={{ err_code[1][code] }}" target="_blank">search forum</a></td>
```

**Review by leecalcote — APPROVED** [review]

**Comment by iArchitSharma** [comment]
@leecalcote, I think I made a small mistake while pushing the code, here is the fix - https://github.com/meshery/meshery/pull/10283

**Pull request merged**